.travis.yml

```diff
@@ -5,8 +5,6 @@ language: erlang
 otp_release:
   - 20.1
   - 20.0
-  - 19.3
-  - 18.3
```

Comment:
Why older OTP version supports are deprecated in this PR?

Reply:
I noticed the following problem with 18.x and 19.x (from Travis https://travis-ci.org/K2InformaticsGmbH/sqlparse/jobs/309585357):

```
The command "chmod +x test/gen_tests.sh" exited with 0.
2.17s$ ./test/gen_tests.sh
16:13:58 Start Test Data Generation
===> Verifying dependencies...
===> Compiling sqlparse
{"init terminating in do_boot",{undef,[{string,replace,["\n        GRANT dw_manager\n           TO sh\n           WITH ADMIN OPTION;\n    ","\"","\\"",all],[]},{sqlparse_generator,'-create_code/1-lc$^159/1-22-',2,[{file,"/home/travis/build/K2InformaticsGmbH/sqlparse/_build/test/lib/sqlparse/test/sqlparse_generator.erl"},{line,4137}]},{sqlparse_generator,create_code,1,[{file,"/home/travis/build/K2InformaticsGmbH/sqlparse/_build/test/lib/sqlparse/test/sqlparse_generator.erl"},{line,4114}]},{sqlparse_generator,create_code,0,[{file,"/home/travis/build/K2InformaticsGmbH/sqlparse/_build/test/lib/sqlparse/test/sqlparse_generator.erl"},{line,236}]},{sqlparse_generator,generate,0,[{file,"/home/travis/build/K2InformaticsGmbH/sqlparse/_build/test/lib/sqlparse/test/sqlparse_generator.erl"},{line,42}]},{init,start_em,1,init terminating in do_boot ()
[]},{init,do_boot,3,[]}]}}
Crash dump is being written to: erl_crash.dump...done
16:14:00 End  Test Data Generation
```